server/src/server.ts

```diff
@@ -594,6 +595,7 @@ function getFixes(textDocument: TextDocument): TextEdit[] {
 	return [];
 }
 
+// @ts-ignore

```

Comment:
😱

Reply:
Oh whoops! Yeah it was complaining about a type mismatch but wasn't due to any of my changes:
```
Types of parameters 'onfulfilled' and 'onfulfilled' are incompatible.
              Types of parameters 'value' and 'value' are incompatible.
                Type 'TextEdit[] | ResponseError<void>' is not assignable to type 'ResponseError<void>'.
                  Type 'TextEdit[]' is missing the following properties from type 'ResponseError<void>': code, data, toJson, name, message
```

I'll remove (and maybe have a little investigate) :smile:

Reply:
Aah, my VS Code wasn't using the workspace TypeScript version (2.8.2) and was instead using 3.8.3. Switching made the problem go away!